iina/HistoryWindowController.swift

```diff
@@ -77,6 +77,10 @@ class HistoryWindowController: NSWindowController, NSOutlineViewDelegate, NSOutl
     outlineView.expandItem(nil, expandChildren: true)
   }
 
+  deinit {
+    NotificationCenter.default.removeObserver(self)
+  }
```

Comment:
Please wrap it with `ObjcUtils.silenced`.

It seems that Cocoa tends to release (leak) the whole memory block after quit and let system GC to handle it, so `deinit` is not being called at a fixed time. In early versions IINA crashes when trying to remove nonexistent observers. I'm not sure it's still needed but it's better to add them for safety reasons.